controller/users.go

```diff
@@ -167,29 +177,57 @@ func ConvertUser(request *goa.RequestData, identity *account.Identity, user *acc
 	var bio string
 	var userURL string
 	var email string
+	var contextInformation workitem.Fields
+
 	if user != nil {
 		fullName = user.FullName
 		imageURL = user.ImageURL
 		bio = user.Bio
 		userURL = user.URL
 		email = user.Email
+		contextInformation = user.ContextInformation
+	}
+
+	// The following will be used for ContextInformation.
+	// The simplest way to represent is to have all fields
+	// as a SimpleType. During conversion from 'model' to 'app',
+	// the value would be returned 'as is'.
+
+	simpleFieldDefinition := workitem.FieldDefinition{
+		Type: workitem.SimpleType{Kind: workitem.KindString},
 	}
+
 	converted := app.Identity{
 		Data: &app.IdentityData{
 			ID:   &id,
 			Type: "identities",
 			Attributes: &app.IdentityDataAttributes{
-				Username:     &userName,
-				FullName:     &fullName,
-				ImageURL:     &imageURL,
-				Bio:          &bio,
-				URL:          &userURL,
-				ProviderType: &providerType,
-				Email:        &email,
+				Username:           &userName,
+				FullName:           &fullName,
+				ImageURL:           &imageURL,
+				Bio:                &bio,
+				URL:                &userURL,
+				ProviderType:       &providerType,
+				Email:              &email,
+				ContextInformation: workitem.Fields{},
 			},
 			Links: createUserLinks(request, uuid),
 		},
 	}
+	for name, value := range contextInformation {
+		if value == nil {
+			// this can be used to unset a key in contextInformation
+			continue
+		}
+		convertedValue, err := simpleFieldDefinition.ConvertFromModel(name, value)
+		if err != nil {
+			log.Error(nil, map[string]interface{}{
+				"err": err,
+			}, fmt.Sprintf("Unable to convert user context field %s ", name))
```

Comment:
Thank you @hectorj2f , I need a session from you on error handling and error expressing ! :)
603b8b13273505bf7e3294901f99d71a470f15e3